data/json/LIXA/LIXA_monsters.json

```diff
@@ -0,0 +1,383 @@
+      }
+    ],
+    "dodge": 7,
+    "vision_day": 50,
+    "vision_night": 3,
+    "harvest": "exempt",
+    "death_function": { "corpse_type": "NO_CORPSE", "message": "The %s becomes part of the endless chamber." },
+    "flags": [ "SEES", "HEARS", "WARM", "NO_BREATHE", "PUSH_MON", "HIT_AND_RUN", "HARDTOSHOOT" ],
+    "armor": { "electric": 1 }
+  },
+  {
+    "id": "LIXA_eigenspectre_3_echo",
+    "type": "MONSTER",
+    "name": { "str": "shifting eigenspectre" },
+    "description": "This distorted figure is a kaliadescope of borders.  Every part of a human, flattened into place, warps wildly from second to second.  The word your mind conjures to describe it is 'rotating', though you're not sure why.",
+    "default_faction": "nether_player_hate",
+    "species": [ "NETHER_EMANATION" ],
+    "bodytype": "human",
+    "volume": "62500 ml",
+    "weight": "2 kg",
+    "hp": 40,
+    "speed": 150,
+    "material": [ "flesh" ],
+    "symbol": "E",
+    "color": "light_gray",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_damage": [ { "damage_type": "pure", "amount": 0, "armor_penetration": 1 } ],
+    "special_attacks": [
+      {
+        "type": "spell",
+        "spell_data": { "id": "phase_skulker_teleport", "hit_self": true },
+        "cooldown": 5,
+        "monster_message": "The %s becomes a feature of the testing chamber."
+      },
+      {
+        "type": "spell",
+        "spell_data": { "id": "LIXA_spawn_copy", "hit_self": true },
+        "cooldown": 15,
+        "monster_message": "The %s is in two places at once, and leaves something of itself behind."
+      },
+      [ "DISAPPEAR", 5 ]
+    ],
+    "dodge": 7,
+    "vision_day": 50,
+    "vision_night": 3,
+    "harvest": "exempt",
+    "death_function": { "corpse_type": "NO_CORPSE", "message": "The %s becomes part of the endless chamber." },
+    "flags": [ "SEES", "HEARS", "WARM", "NO_BREATHE", "PUSH_MON", "HIT_AND_RUN", "HARDTOSHOOT" ],
+    "armor": { "electric": 1 }
+  },
+  {
+    "id": "LIXA_eigenspectre_4",
+    "type": "MONSTER",
+    "name": { "str": "titanic eigenspectre" },
+    "description": "There is somehow more of this figure than the others.  Every side of every surface is on display, but each of those seems like the thin edge of a larger shape.  There is an eternity here, where a scientist used to be.",
+    "default_faction": "nether_player_hate",
+    "species": [ "NETHER_EMANATION" ],
+    "bodytype": "human",
+    "volume": "72500 ml",
+    "weight": "2 kg",
+    "hp": 200,
+    "speed": 90,
+    "material": [ "flesh" ],
+    "symbol": "E",
+    "color": "light_gray",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 8,
+    "melee_damage": [ { "damage_type": "pure", "amount": 0, "armor_penetration": 1 } ],
+    "special_attacks": [
+      { "type": "smash", "throw_strength": 50, "cooldown": 20 },
```

Comment:
Does smash require all of the following fields?
`{ "id": "smash", "move_cost": 100, "cooldown": 8, "min_mul": 0.8, "throw_strength": 50 },`
Or at least a move cost?

Reply:
Genuinely no idea; I think I picked this up exactly in the middle of a recent change to the special attacks docs? I originally tried to use the hardcode, but that is one of the things that got shaken up by the Concrete Isn't Real kerfuffle.